### PR TITLE
Clean up old grade details files and references

### PR DESCRIPTION
**Pull request by MariagraziaAlastra**
**Reviewer:** @andrey-p 

## Description:
Removing unused legacy code (view and template) for the grade details subview.

## Steps to test this PR:
1. Build and reload extension
2. Make sure the grade details work as expected and nothing is missing

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: shared/js/ui/templates/grade-details.es6.js [hunk 1]
```diff
@@ -1,98 +0,0 @@
-const bel = require('bel')
-const header = require('./shared/sliding-subview-header.es6.js')
-const siteRating = require('./shared/site-rating.es6.js')
-const siteRatingExplainer = require('./shared/site-rating-explainer.es6.js')
```
Comment (andrey-p) [inline shared/js/ui/templates/grade-details.es6.js]: The `site-rating` and `site-rating-explainer` templates can also go.

File: shared/js/ui/templates/grade-details.es6.js [hunk 2]
```diff
@@ -1,98 +0,0 @@
...
-const tosdrMessages = {'A': 'Good', 'B': 'Mixed', 'C': 'Poor', 'D': 'Poor'}
-
-module.exports = function () {
-  if (!this.model) {
-    return bel`<section class="sliding-subview
-    sliding-subview--has-fixed-header">
-      ${header('Grade Details')}
-    </section>`
-  } else {
-    return bel`<div class="site-info site-info--details card card--no-top-margin">
```
Comment (andrey-p) [inline shared/js/ui/templates/grade-details.es6.js]: The styles for `site-info--details` and `site-info--details__explainer` can also go. (They live in popup.scss.)

File: shared/js/ui/views/site.es6.js [hunk 3]
```diff
@@ -1,9 +1,7 @@
 const Parent = window.DDG.base.View
 const GradeScorecardView = require('./../views/grade-scorecard.es6.js')
-const GradeDetailsView = require('./../views/grade-details.es6.js')
 const TrackerNetworksView = require('./../views/tracker-networks.es6.js')
 const PrivacyPracticesView = require('./../views/privacy-practices.es6.js')
-const gradeDetailsTemplate = require('./../templates/grade-details.es6.js')
```
Comment (andrey-p) [inline shared/js/ui/views/site.es6.js]: These are still referenced elsewhere in this file.

File: shared/js/ui/templates/grade-details.es6.js [hunk 4]
```diff
@@ -1,98 +0,0 @@
...
-        this.model.site.siteRating,
-        this.model.site.isWhitelisted)}
-      <h2 class="site-info__https-status padded border--bottom">
-        ${httpsMsg(this.model.site.httpsState)}
-        <div class="float-right"></div>
-      </h2>
-      <h2 class="site-info__tosdr-status">
-        ${tosdrMsg(this.model.site.tosdr)}
-      </h2>
-      <p class="site-info__tosdr-msg padded border--bottom">
```
Comment (andrey-p) [inline shared/js/ui/templates/grade-details.es6.js]: Some of the `site-info__[something]` classes can also be removed, I think.

**Comment by MariagraziaAlastra** [comment]
@andrey-p Removed all the extra stuff! I think the site-info__tosdr styles were the only ones used by the old grade details

**Comment by andrey-p** [comment]
Awesome, thank you! :+1:

**Pull request merged**